### PR TITLE
🐛 fixed issue invalid json response body when call revokeAuthorizationToken function

### PR DESCRIPTION
**Pull request by dauden**
fixed issue invalid json response body

Error details:

FetchError: invalid json response body at https://appleid.apple.com/auth/revoke reason: Unexpected end of JSON input | CorrelationId: 507783AA-58EA-41EC-9984-0ABD52701AE3_1655970089157 FetchError: invalid json response body at https://appleid.apple.com/auth/revoke reason: Unexpected end of JSON input

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ const getAuthorizationToken = async (
   return fetch(url.toString(), {
     method: 'POST',
     body: params,
-  }).then((res) => res.json());
+  }).then((res) => (typeof res?.json === 'function' ? res.json() : res));
```
Comment (A-Tokyo) [inline src/index.js:223]: @dauden  Should we do this for the other functions as well? eg: revokeAuthorizationToken

Comment (dauden) [reply src/index.js]: Yeah! my bad, I will push change now

Comment (dauden) [reply src/index.js]: @A-Tokyo I have changed all of the function 

thanks

**Review by A-Tokyo — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (A-Tokyo) [reply src/index.js]: My pleasure, merging now

**Pull request merged**